docs/content/getting-started/reference-apps/aws-sqs/snippets/app.bicep

```
@@ -0,0 +1,69 @@
+import aws as aws
+import radius as radius
+
+param location string = 'global'
+param environment string
+param app_name string
```

In your instructions you say:

```
The AWS SQS Queue name must follow the [Queue name conventions](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/quotas-queues.html).
```

However your parameter is called `app_name`, as you're using this parameter as both the application's name and a seed to generate the SQS resource names.

I'd recommend changing this parameter to `sqs_name`, and hardcoding your app name to something like `'sqs-reference-app'`

```suggestion
param sqs_name string
```